spec/temporal.html

```diff
@@ -193,5 +215,22 @@ <h1>SystemDateTime ( [ _temporalTimeZoneLike_ [ , _calendarLike_ ] ] )</h1>
         1. Return ? GetTemporalDateTimeFor(_timeZone_, _instant_, _calendar_).
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-temporal-systemzoneddatetime" aoid="SystemZonedDateTime">
+      <h1>SystemZonedDateTime ( [ _temporalTimeZoneLike_ [ , _calendarLike_ ] ] )</h1>
+      <emu-alg>
+        1. If _temporalTimeZoneLike_ is *undefined*, then
+          1. Let _timeZone_ be ? SystemTimeZone().
+        1. Else,
+          1. Let _timeZone_ be ? ToTemporalTimeZone(_temporalTimeZoneLike_).
+        1. If _calendarLike_ is *undefined*, then
+          1. Let _calendar_ be ? GetISO8601Calendar().
+        1. Else,
+          1. Let _calendar_ be ? ToTemporalCalendar(_calendarLike_).
+        1. Let _ns_ be the approximate current UTC date and time, in nanoseconds since the epoch.
```

Comment:
There's an `<emu-note>` about this step in SystemInstant which should apply to this operation as well. Rather than duplicating the note, I'd suggest creating an operation SystemUTCEpochNanoseconds (since we have one in the polyfill anyway) that goes something like this:
```html
<emu-alg>
  1. Let _ns_ be the approximate current UTC date and time, in nanoseconds since the epoch.
  1. Assert: ! ValidateInstant(_ns_) is *true*.
  1. Return _ns_.
</emu-alg>
<emu-note>
  <p>The precision of the result depends on the host. In particular, web browsers artificially
  limit it to prevent abuse of security flaws (e.g., Spectre) and to avoid certain methods of
  fingerprinting.</p>
</emu-note>
```
(and use the operation both here and in SystemInstant, and remove the note from SystemInstant.)